type-definitions/immutable.js.flow

```diff
@@ -449,15 +478,29 @@ declare class List<T> extends IndexedCollection<T> {
     mapper: (value: T, index: number, iter: this) => ESIterable<M>,
     context?: any
   ): List<M>;
+
+  filter(
+    predicate: (value: T, index: number, iter: this) => mixed,
+    context?: any
+  ): List;
```

Comment:
perhaps make this an explicit `List<any>` so it's clear that this was moved here to define use of `<any>`?